src/components/WwwFrame/Menus/CustomSecondaryMenu.vue

```diff
@@ -0,0 +1,216 @@
+<template>
+	<div>
+		<secondary-menu class="show-for-large">
+			<ul class="row">
+				<li v-for="{name, routerLink, url, eventTracking} in menuItems" :key="name">
+					<router-link
+						:to="routerLink"
+						v-kv-track-event="eventTracking"
+						v-if="routerLink"
+					>
+						{{ name }}
+					</router-link>
+					<a v-else :href="url" v-kv-track-event="eventTracking">{{ name }}</a>
+				</li>
+			</ul>
+		</secondary-menu>
+		<div class="mobile-nav hide-for-large">
+			<button
+				@click="toggle"
+				aria-controls="secondary-menus-combo-nav"
+				:aria-expanded="open ? 'true' : 'false'"
+			>
+				<span>Menu</span>
+				<kv-icon name="small-chevron-mobile" />
+			</button>
+			<kv-expandable easing="ease-in-out">
+				<div
+					id="secondary-menu-combo-nav"
+					v-show="open"
+					class="kv-expandable-pane sec-ter-combo-nav"
+					:aria-hidden="open ? 'false' : 'true'"
+				>
+					<ul>
+						<li v-for="{name, routerLink, url, eventTracking} in menuItems" :key="name">
+							<router-link
+								:to="routerLink"
+								v-kv-track-event="eventTracking"
+								v-if="routerLink"
+							>
+								{{ name }}
+							</router-link>
+							<a v-else :href="url" v-kv-track-event="eventTracking">{{ name }}</a>
+						</li>
+					</ul>
+				</div>
+			</kv-expandable>
+		</div>
+	</div>
+</template>
+
+<script>
+import _get from 'lodash/get';
+import {
+	onBodyTouchstart,
+	offBodyTouchstart,
+	isTargetElement,
+} from '@/util/touchEvents';
+import customSecondaryMenuQuery from '@/graphql/query/customSecondaryMenuQuery.graphql';
+import KvIcon from '@/components/Kv/KvIcon';
+import SecondaryMenu from '@/components/WwwFrame/SecondaryMenu';
+import KvExpandable from '@/components/Kv/KvExpandable';
+
+export default {
+	components: {
+		KvIcon,
+		SecondaryMenu,
+		KvExpandable,
+	},
+	inject: ['apollo'],
+	data() {
+		return {
+			open: false,
+			usingTouch: false,
+		};
+	},
+	props: {
+		menuItems: {
+			type: Array,
+			required: true,
+		}
+	},
+	apollo: {
+		query: customSecondaryMenuQuery,
+		preFetch: true,
+		result({ data }) {
+			this.usingTouch = _get(data, 'usingTouch');
```

Comment:
@ksekhri in case this value isn't provided or is ill formed will it default to the initial value? if this is done implicitly i will prefer to do it explicitly.

Reply:
I did a test and when the value is missing `_get` returns undefined which implicitly is casted to false. I believe in the situations where this fails, false would work so I've replaced that line with:
```
this.usingTouch = !!_get(data, 'usingTouch');
```